### PR TITLE
Enhancement/post_officer_history_reports

### PR DESCRIPTION
**Pull request by ayyubibrahimi**

### CODE REVIEW

File: fuse/post_officer_history.py [hunk 1]
```diff
@@ -34,15 +34,16 @@ def fuse_events(pprr):
 
 
 if __name__ == "__main__":
-    post = pd.read_csv(deba.data("match/post_officer_history.csv"))
+    post = pd.read_csv(deba.data("clean/post_officer_history.csv"))
```
Comment (pckhoi) [inline fuse/post_officer_history.py:37]: Any reason for discarding the match result?

Comment (ayyubibrahimi) [reply fuse/post_officer_history.py]: I used the wrong decision score for match. Fixed.

**Comment by ayyubibrahimi** [comment]
@pckhoi - pretty confused by the `process-data` failure as those columns `officer_name_[1-6]` no longer exist when they are dropped after `dfa` and `dfb` are concatenated.

**Comment by ayyubibrahimi** [comment]
@pckhoi as mentioned in PR#329, I've appended the metadata to the `fuse` script. I will gladly make changes based on your feedback. Thanks.

**Comment by pckhoi** [comment]
Are you not able to reproduce that error on local? I wonder what is different.

**Comment by pckhoi** [comment]
And also please investigate why the diff output on Github is so hard to read. This makes it impossible to review. Probably Windows CR/LF again.

**Comment by ayyubibrahimi** [comment]
Output isn't easier to read. Will investigate further.

**Comment by ayyubibrahimi** [comment]
@pckhoi - Please review the process-data failure again. Very strange.

It's another issue that can't be reproduced on local. More specifically, it's an issue that I explicitly addressed in the `clean_hire_date` function in order for me to add `.pipe(clean_dates, ["hire_date", "left_date"])` to the df.

**Comment by pckhoi** [comment]
Please ensure that you are running Python 3.9, and all the packages match by running `pip install -r requirements.txt`. Also remove the raw data files and `dvc pull` afresh. This way your environment should match and the error should be reproduced.

**Comment by ayyubibrahimi** [comment]
I'm running python 3.9 and packages were installed with `pip install -r requirements.txt`. I will now remove raw datafiles and run `dvc pull`.

**Comment by ayyubibrahimi** [comment]
Closing this duplicate PR.

**Pull request closed without merging**